src/Components/programs/Program.js

```diff
@@ -15,23 +18,29 @@ export const Program = props => {
   const onPress = () => {
     const firstTodoTraining = trainings.find(training => !programCompleteTrainings.includes(training));
     if (firstTodoTraining !== undefined) {
-      props.navigation.navigate('TrainingPage', { training: firstTodoTraining, program: props.program });
+      props.navigation.navigate('ProgramPage', { program: props.program });
     }
   };
 
   const width = `${(completeTrainingsCount * 100) / trainings.length}%`;
-  const completeStyle = completeTrainingsCount === trainings.length ? styles.complete : null;
   return (
-    <TouchableOpacity style={[styles.program, completeStyle]} onPress={onPress}>
-      <Text style={styles.programTitle}>{title}</Text>
-      <Text style={styles.completion}>
-        {completeTrainingsCount}/{trainings.length} trainings
-        {completeTrainingsCount === trainings.length && ' 👍🎉'}
-      </Text>
-      <View style={styles.progressBar}>
-        <View style={[StyleSheet.absoluteFill, styles.fillProgressBar, { width }]} />
-      </View>
-    </TouchableOpacity>
+    <ListItem>
+      <TouchableOpacity style={styles.program} onPress={onPress}>
+        <View style={styles.presentationContainer}>
+          <Text style={styles.programTitle}>{title}</Text>
+          <Text style={styles.completion}>
+            {completeTrainingsCount}/{trainings.length} trainings
```

Comment:
Its detail and won't happen because programs will always have several trainings I guess, but I don't like this training**s** when there is only one program

Reply:
I guess we could include pluralization management, but maybe later ? (june version)